trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapUkefExposure.js

```diff
@@ -33,7 +36,8 @@ const mapUkefExposure = async (facilityTfm, facility) => {
 
         // sets new exposure value based on amendment value
         formattedUkefExposure = formattedNumber(ukefExposureValue);
-        ukefExposureCalculationTimestampValue = getUnixTime(new Date());
+        // converts from seconds unix timestamp to one with milliseconds
+        ukefExposureCalculationTimestampValue = fromUnixTime(ukefExposureCalculationTime).valueOf();
```

Comment:
@abhi-markan  I need your input here: do we want to use `*1000` or `valueOf()`? alternatively, the `submitted` at value is generated using date-fns. Do we want to use `Date.now()` instead?

Reply:
just to add, for the second option for changing the generated value may mean changing quite a few files to display ms properly

Reply:
noted. 
@abhi-markan if you want this standardised, then it needs to be implemented before we go live with amendments

Reply:
1. What format of value does the variable `ukefExposureCalculationTimestampValue` will have for both the conditions.
2. We need to ensure both the values of above variable in ternary operator both are consistent i.e. EPOCH without ms.

Reply:
mongodb value is in epoch milliseconds (none amendment based).  Amendment is in epoch seconds for both sides of the ternary operator

Reply:
So just to confirm values being saved in DB are both saved from `Date` function?

Reply:
done - and to answer your question @abhi-markan ukefExposureCalculationTimestamp is saved in tfm-facilities from Date function whereas submittedAs is from getUnixTime from date-fns